lib/utils.js

```diff
@@ -98,12 +97,17 @@ const utils = {
    * If we are in a PR build, just grab the sha from env, if not
```

Comment:
From your comment on slack:
> It now looks up the PR via the Github API to determine the merge SHA of the PR rather than using the environment variable `TRAVIS_COMMIT` (which it sounds like doesn’t update when closing/re-opening a PR)

It leads me to believe that this comment in the code might not be accurate anymore?